include/glow/Graph/Node.h

```diff
@@ -45,30 +45,43 @@ struct NodeValue {
   Node *node_{nullptr};
   /// Specifies the node result number to use.
   unsigned resNo_{0};
+  /// Is this value an operand of some instruction or just a temporary object
+  /// flying around.
+  bool isOperand_{false};
```

Comment:
This bool isOperand_ will result in consuming an additional word probably. I wonder if we should try to use `llvm::PointerIntPair`. But it is purely an optimization to reduce memory.  No real need to think about it in this PR.